pkg/controller/allowlist/allowlist_controller.go

```diff
@@ -37,19 +45,38 @@ func Add(mgr manager.Manager, status *statusmanager.StatusManager, c cnoclient.C
 	return add(mgr, newReconciler(mgr, status, c))
 }
 
-func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager, c cnoclient.Client) reconcile.Reconciler {
+func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager, c cnoclient.Client) *ReconcileAllowlist {
 	return &ReconcileAllowlist{client: c, scheme: mgr.GetScheme(), status: status}
 }
 
-func add(mgr manager.Manager, r reconcile.Reconciler) error {
+func add(mgr manager.Manager, r *ReconcileAllowlist) error {
 	c, err := controller.New("allowlist-controller", mgr, controller.Options{Reconciler: r})
 	if err != nil {
 		return err
 	}
-	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{})
-	if err != nil {
+
+	// watch for changes in all configmaps in our namespace
+	cmInformer := v1coreinformers.NewConfigMapInformer(
+		r.client.Default().Kubernetes(),
+		names.MULTUS_NAMESPACE,
+		0, // don't resync
+		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
+
+	r.client.Default().AddCustomInformer(cmInformer) // Tell the ClusterClient about this informer
+
+	if err := c.Watch(&source.Informer{Informer: cmInformer},
```

Comment:
you can simply return here

Reply:
Ok. I need to rebase the PR, so I'll update the code and retest

Reply:
Done